timemachine/fe/free_energy.py

```diff
@@ -1099,7 +1094,7 @@ def replica_from_samples(traj: Trajectory) -> CoordsVelBox:
         hrex, samples_by_state_iter = hrex.sample_replicas(sample_replica, replica_from_samples)
         log_q_kl = compute_log_q_matrix(hrex.replicas)
         hrex, fraction_accepted_by_pair = hrex.attempt_neighbor_swaps_fast(
-            neighbor_pairs, log_q_kl, n_swap_attempts_per_iter
+            neighbor_pairs, log_q_kl, n_swap_attempts_per_iter, md_params.seed
```

Comment:
Should this do `md_params.seed + iteration` to avoid generating the same sequence every time?

Reply:
Yes, I think so! Good catch.